cdk/package.json

```diff
@@ -42,5 +42,9 @@
     "source-map-support": "^0.5.16",
     "stream-chain": "^2.2.5",
     "stream-json": "^1.7.4"
+  },
+  "engines": {
+    "npm": ">=8.0.0",
+    "node": ">=16.0.0"
```

Comment:
This should be <18, since 18 introduces breaking changes that make importing `process` fail.

Reply:
Done!